### PR TITLE
logic implementation of the last video viewed by the user in the dash…

### PR DESCRIPTION
**Pull request by Daniielmayo**
logic implementation of the last video viewed by the user in the dashboard component

### CODE REVIEW

**Review by DanielCalleCO — APPROVED** [review]
Nicely DONE! Remember that I just reviewed your code, not the functionality or the previews, so please ensure you get the approval from anyone who deploys your branch locally before merging.

**Review by SthefaniaEcheverry — APPROVED** [review]

**Pull request merged**
